static/app/components/feedback/widget/useFeedbackWidget.tsx

```diff
@@ -32,15 +30,12 @@ export default function useFeedbackWidget({buttonRef, messagePlaceholder}: Props
 
     if (buttonRef) {
       if (buttonRef.current) {
-        const widget = feedback.attachTo(buttonRef.current, options);
-        return () => {
-          feedback.removeWidget(widget);
-        };
+        return feedback.attachTo(buttonRef.current, options);
       }
     } else {
       const widget = feedback.createWidget(options);
       return () => {
-        feedback.removeWidget(widget);
+        widget.removeFromDom();
```

Comment:
This is working now with beta 5

example from the Releases page
![SCR-20240430-koxr](https://github.com/getsentry/sentry/assets/187460/6b5deb22-353a-484c-a99e-1f05073d8338)